here/traffic/sql/move_tables.sql

```diff
@@ -0,0 +1,35 @@
+-- Move data from here.ta_* to the newly structured here.ta_*
+-- while performing separation of tx column to dt (Date) and tod (time)
+-- loop through each table, move data to new table, then truncate the old one
+--Loops through the years and then months for which we currently have data in order to create a partitioned table for each month
+
```

Comment:
This needs to be done in Python, because otherwise committing won't happen until the loop is finished and it's going to be a big deal